docs/user/ppl/functions/datetime.rst

```diff
@@ -15,28 +15,32 @@ Description
 >>>>>>>>>>>
 
 Usage: adddate(date, INTERVAL expr unit)/ adddate(date, expr) adds the time interval of second argument to date; adddate(date, days) adds the second argument as integer number of days to date.
+If first argument is TIME, today's date is used; if first argument is DATE, time at midnight is used.
```

Comment:
Can you update the first line to: 
```
Usage: adddate(date, INTERVAL expr unit)/ adddate(date, date) adds the interval of second argument to date; adddate(date, days) adds the second argument as integer number of days to date.
```
Time interval might be confusing, since it can be a 'date'.  And the variable name should be consistent in the second sentence.